docs/ipdk-dpdk.md

```diff
@@ -53,7 +53,8 @@ sudo mkdir -p /etc/stratum/
 sudo mkdir -p /var/log/stratum/
```

Comment:
Given the number of (manually-entered) commands, wouldn't it be more convenient for the customer if we put them in a bash script in the `scripts` directory?

Reply:
Totally agree, will take it as a separate PR